src/routes/+page.svelte

```diff
@@ -307,6 +307,27 @@
 	</div>
 </Section>
 
+<!-- About us -->
+<Section id="about-us" class="max-sm:child:!mx-8">
```

Comment:
I still don't like it

It's not required, the text is tall anyway and the only thing this does is remove a bit of margins to make the text 1 line shorter. The main problem relies on the fact that it's not consistent with the rest of the page, and we don't really get why this section is the only one like this.

If you _really_ want it, apply this margin change **only** on the div (L316) but I completely disagree with changing that margin on the Section title too.

Reply:
So i prefer to reduce the padding on each section on mobile to make it consistent.

Reply:
Really? Do you think this section is so important that we should change _**all of them**_? I was not expecting that but if you **really** think it **has to** be done, then make a separate PR for this...

Reply:
Another solution would be to reduce the text. On mobile is really fat. Like you said this section isn't the most important one

Reply:
I'm 100% for reducing text! Much more reasonable solution haha